src/processor/tasks.py

```diff
@@ -34,33 +34,29 @@ def pull_metrics():
     db.session.add(scrape)
     db.session.commit()
 
-    try:
```

Comment:
This section was just me removing the `try/except` (since `Sentry` is more effective), so feel free to ignore it.